packages/client/hmi-client/src/types/Types.ts

```diff
@@ -621,6 +621,23 @@ export interface EntitySimilarityResult {
     similarity: number;
 }
 
+export interface NotificationEvent {
```

Comment:
These don't seem to be models required in the client... I appreciate this PR might not be the right place for this sort of clean up.  @YohannParis we should probably think about what models are actually shared with the UI and which should not be.